README.md

```diff
@@ -56,10 +56,12 @@ down to the following steps:
    Cargo workspace.
 2. Create a `flake.nix` file with nixpkgs, [cargo2nix] and [nixpkgs-mozilla] inputs
```

Comment:
#178 switches to Oxalica's overlay by default.   Want me to merge the changes into this branch?

I appreciate the flake example.

I can fix up the CI branch pointer and merge this if you're ready to have it all in master.

Reply:
Sure! I also successfully use this branch in my projects. Let's dot it.

Reply:
Sure! But I think that there should be an option to use nixpkgs-mozilla, because in some cases relying on third-party sha256 is not an option. Given that, I thing having `alternative-overlay` example using nixpkgs-mozilla should be enough.

Reply:
Considering what would count as "some cases."  What can the moz overlay do that the Oxalica one can't, because the github actions are likely scheduled right after rustup updates.  Can moz provide a toolchain that rustup cannot?

Reply:
The point of moz for such cases is that you don't rely on third-party provided sha256. AFAIK, with oxalica's you can't force your own sha256 check. However, this check can be implemented in the Oxalica's overlay.

Reply:
I'll put the argument back in.  We can't pass it into the Oxalica overlay though.  Moz will be available but unmaintained.

In reality it's all trust-on-first-use and riding on top of Rust's content verification in the end.  Oxalica overlay might be better off in nix community repos.  Obviously anyone can pin and anyone can verify the sha256 manually and independently.